articles/iot-suite/iot-suite-what-is-azure-iot.md

```diff
@@ -27,5 +27,5 @@ Now that you have learned about the typical IoT Architecture, explore the differ
 To find out more about the individual Azure IoT services, see:
 
 * [What is Azure IoT Suite?](iot-suite-what-are-preconfigured-solutions.md)
-* [What is Microsoft IoT Central?](https://www.microsoft.com/internet-of-things/iot-central-saas-solutions)
+* [What is Microsoft IoT Central?](https://docs.microsoft.com/en-us/microsoft-iot-central/overview-iot-central)
```

Comment:
Can you change this to be: https://docs.microsoft.com/microsoft-iot-central/overview-iot-central

Removing the locale identifier means that it can redirect to a page in the user's language.
Thank you!